actionpack/lib/action_dispatch/middleware/templates/rescues/missing_exact_template.html.erb

```diff
@@ -1,19 +1,23 @@
 <header role="banner">
-  <h1>No template for interactive request</h1>
+  <h1>No view template for interactive request</h1>
 </header>
 
 <main id="container">
   <h2><%= h @exception.message %></h2>
 
-  <p class="summary">
-    <strong>NOTE!</strong><br>
-    Unless told otherwise, Rails expects an action to render a template with the same name,<br>
-    contained in a folder named after its controller.
-
-    If this controller is an API responding with 204 (No Content), <br>
-    which does not require a template,
-    then this error will occur when trying to access it via browser,<br>
-    since we expect an HTML template
-    to be rendered for such requests. If that's the case, carry on.
-  </p>
+  <div class="summary">
+    <p>
+      <strong>NOTE:</strong>Rails usually expects a controller action to render a view template with the same name. 
+    </p>
+    <p>
+      For example, a <code>BooksController#index</code> action defined in <code>app/controller/books_controller.rb</code> should have a corresponding view template 
+      in a file named <code>app/views/books/index.erb.html</code>.
```

Comment:
I like this a lot! I wonder (a non-blocking idea) if we could pass through the controller and action we were expecting from the implicit render in order for this message to include the actual filename they probably intend to create.

Reply:
I thought about it for a second and then decided to keep the scope down for this PR. but I do like the idea too. As a follow-up?

I think it’s in line with recent improvements to migration related error feedback with the “migrate” button.

Reply:
Shouldn't this be `app/views/books/index.html.erb` - I've never seen this inverted, and may be especially confusing as this is a change aimed at beginners?

Reply:
Good catch. I fixed it in cfa9c9f89184cfb14bdfd8fce58a1b630142007a